File: pilhas-filas/python/queue1_test.py
from queue1 import Queue1

queue = Queue1()

print("queue.count():", queue.count())
print("queue.is_empty():", queue.is_empty())

queue.add("Maria")
queue.add("Joao")
queue.add("Ana")
print("Adicionados: Maria, Joao, Ana")

print("queue.count():", queue.count())
print("queue.is_empty():", queue.is_empty())
print("queue.peek():", queue.peek())

print("Itens:")
while not queue.is_empty():
    print(queue.remove())

File: pilhas-filas/java/src/Queue1Test.java
public class Queue1Test {
    public static void main(String[] args) {
        Queue1<String> queue = new Queue1<>();

        System.out.println("queue.count(): " + queue.count());
        System.out.println("queue.isEmpty(): " + queue.isEmpty());

        queue.add("Maria");
        queue.add("Joao");
        queue.add("Ana");
        System.out.println("Adicionados: Maria, Joao, Ana");

        System.out.println("queue.count(): " + queue.count());
        System.out.println("queue.isEmpty(): " + queue.isEmpty());
        System.out.println("queue.peek(): " + queue.peek());

        System.out.println("Itens:");
        while (!queue.isEmpty()) {
            System.out.println(queue.remove());
        }
    }
}
